src/main/java/org/commcare/xml/CaseXmlParser.java

```diff
@@ -204,6 +204,18 @@ private void updateCase(Case caseForBlock, String caseId) throws InvalidStructur
                 case "external_id":
                     caseForBlock.setExternalId(value);
                     break;
+                case "category":
+                    caseForBlock.setCategory(value);
+                    break;
+                case "state":
+                    caseForBlock.setState(value);
+                    break;
+                case "patient_type":
```

Comment:
You should remove the `patient_type` and `current_status` aliasing, here, otherwise the system won't set the original values.

We still want `patient_type` to update the `patient_type` key in the case, we just also want it to populate a totally separate index.